api.go

```diff
@@ -492,8 +492,8 @@ func (d *Context) GetLength(index int) int {
 }
 
 // See: http://duktape.org/api.html#duk_get_lstring
-func (d *Context) GetLstring(index int, outLen int) string {
-	if s := C.duk_get_lstring(d.duk_context, C.duk_idx_t(index), (*C.duk_size_t)(unsafe.Pointer(&outLen))); s != nil {
+func (d *Context) GetLstring(index int, outLen *int) string {
```

Comment:
Strange, it looks like you might need to pull down the latest changes: https://github.com/mosolovsa/go-duktape/commit/318b65661207e8d88941cee476518009e39ab321

That commit fixed that issue.